Archiver.py

```diff
@@ -1,4 +1,30 @@
 
+import os.path
+
+
+def read_file(file_path):
+    file_path = os.path.abspath(file_path)
+    file_name = os.path.basename(file_path).split(".")[0]
+    try:
+        with open(file_path, "r") as input_file:
+            out_string = input_file.readline()
+    except FileNotFoundError:
+        print("So file is not found, try again")
+        out_string = None
+        file_name = None
+    return out_string, file_name
+
+
+def write_file(final_string, input_file_name, zipping=False):
```

Comment:
Zipping = True means we are decompressing the file?
This makes not much sense, to be honest.
The opposite naming makes much more sense. Like, if 'zipping' = True  - we are compressing

Reply:
Yes, my English is bad and I'm not good in names. But opposite naming is really wrong there.